docs/content/en/docs/reference/clusterspec/cni.md

```diff
@@ -0,0 +1,173 @@
+---
+title: "cni plugin configuration"
+linkTitle: "cni"
+weight: 20
+description: >
+EKS Anywhere cluster yaml cni plugin specification reference
+---
+
+### Specifying CNI Plugin in EKS Anywhere cluster spec
+
+EKS Anywhere currently supports two CNI plugins: Cilium and Kindnet. Only one of them can be selected
+for a cluster, and the plugin cannot be changed once the cluster is created.
+Up until the 0.7.x releases, the plugin had to be specified using the `cni` field on cluster spec.
+Starting release 0.8, the plugin should be specified using the new `cniConfig` field as follows:
+
+- For selecting Cilium as the CNI plugin:
+```yaml
+apiVersion: anywhere.eks.amazonaws.com/v1alpha1
+kind: Cluster
+metadata:
+  name: my-cluster-name
+spec:
+  clusterNetwork:
+    pods:
+      cidrBlocks:
+      - 192.168.0.0/16
+    services:
+      cidrBlocks:
+      - 10.96.0.0/12
+    cniConfig:
+      cilium: {}
+```
+EKS Anywhere selects this as the default plugin when generating a cluster config.
+
+- Or for selecting Kindnetd as the CNI plugin:
```

Comment:
Same issue here. Indent lines 36 through 51 to align with the text under the bullet.